autoendpoint/src/error.rs

```diff
@@ -59,12 +59,21 @@ pub enum ApiErrorKind {
     #[error(transparent)]
     VapidError(#[from] VapidError),
 
+    #[error(transparent)]
+    Uuid(#[from] uuid::Error),
```

Comment:
_(mutters something about preferring errors be discretely named values like `ApiErrorKind::UuidError`, but realizes it's probably not worth fighting over.)_

Reply:
I was going back and forth about this. It's nice to be clear that it's an error type, but `ApiErrorKind::Uuid` is pretty clear that it's an error. Adding `Error` on to the end of every variant seems pretty redundant.

Reply:
yeah, like i said, not really worth fighting over, particularly since this instance of `Uuid` (or `Database`) should never appear without it's type.